### DISCUSSION TITLE
In users section see the users depending on the user group they are part of

### DISCUSSION THREAD

**Discussion by alina-tincas** [umbraco/Umbraco-CMS#12789] [Features and ideas]
Currently no matter what user group you are in, if you have access to the users section, you can see all users.
**Scenario**:
- create a user group called A an a user group called B (make sure they have permission to access the users sections)
- invite one user to user group A and another user to user group B
- then both users invited to group A and B can see all the users, no matter the user group.

Due to sensitive data group A /B should see users that are part only from group A, and not all user groups users. 

**Expected scenario**:
- users from user group A should see only users from group A and users from user group B should see users only from user group B.

**Comment by skttl** [comment]
In addition to this, it would be great if you could make a user group able to invite new users, without giving them all admin rights.

Right now, you need to give the user access to the user section, but that also allows the user to add any other section to their own profile.
